src/components/runtime-common.js

```diff
@@ -128,6 +129,7 @@ export const SaveFilesHelpRStudio = () => {
     p([
       'If you want to save files permanently, including input data, analysis outputs, installed packages or code in your session, ',
       h(Link, {
+        'aria-label': 'RStudio save help',
```

Comment:
Since this will be read as a sentence, 'move them to the workspace bucket' makes more sense to me than this label. If anything, we might insert the text 'learn how to' before the link, or something similar.
```suggestion
```